jobserver/models/outputs.py

```diff
@@ -11,6 +11,38 @@ def absolute_file_path(path):
     return abs_path
 
 
+class PublicRelease(models.Model):
+    """
+    A Release for a Workspace which has been, or will be made publicly available
+    """
+
+    class Statuses(models.TextChoices):
```

Comment:
Do'h, I now realise I didn't understand what you meant in your PR comment earlier!